src/clj_gatling/progress_tracker.clj

```diff
@@ -1,17 +1,18 @@
 (ns clj-gatling.progress-tracker
   (:require [clj-gatling.simulation-runners :as runners])
-  (:import (java.util.concurrent Executors TimeUnit)))
+  (:import  (java.time LocalDateTime)
+            (java.util.concurrent Executors TimeUnit)))
 
 (defn create-console-progress-tracker []
   (let [finished? (atom false)]
     (fn [{:keys [progress sent-requests total-concurrency]}]
       (when-not @finished?
         (let [progress-percent (int (* 100 progress))]
-          (print (str "Progress: " progress-percent "%, concurrency: " total-concurrency " , sent requests: " sent-requests "\r"))
+          (println (str "Progress: " progress-percent "%, concurrency: " total-concurrency " , sent requests: " sent-requests))
```

Comment:
This is using `print` because the idea is to write to same line so that it works as console progress bar.

Reply:
Ahha, that makes sense. I'll change it back. I was running it through the REPL, so I was getting output that was harder to read because it didn't overwrite and everything was one one line, but if the intention is CLI-based fair enough 👍

Reply:
Reverted.